src/main/java/com/carpool/car_pool/services/RideOfferService.java

```diff
@@ -91,4 +90,15 @@ public RideOfferResponse editRideOfferDetail(EditRideOfferRequest editRideofferR
 
 
     }
+
+    public void deleteRideOffer(long id, UserEntity user) {
```

Comment:
All delete methods should be transactional so in case of error all changes to database would be discarded